examples/bert/bert_tpe_config_classifier.py

```diff
@@ -0,0 +1,11 @@
+name = "bert_classifier"
```

Comment:
replace `tpe` w/ `hypertuning` in file names

Reply:
Done in e175fa6